README.md

```diff
@@ -0,0 +1,29 @@
+# How To Build And Run PyTorch For TPU
+
+To build:
+
+* Build PyTorch from source, following the regular [instructions](https://github.com/pytorch/pytorch#from-source).
+* Clone this repository in the root folder of the PyTorch sources used for the previous step.
```

Comment:
we shouldn't need to clone the repo in the root folder of PyTorch, and build pytorch from source. This should work out with PyTorch binary installs.
We can move in that direction in a follow-up PR, I can help with it.

Basically, the headers you are pulling from tmp_install, or from torch/csrc should be already installed to `site-packages/torch/lib/include`

Reply:
Thanks, that would be a lot better.